internal/flavors/publisher.go

```diff
@@ -19,12 +19,19 @@ package flavors
 
 import (
 	"context"
+	"fmt"
 	"time"
 
 	"github.com/elastic/beats/v7/libbeat/beat"
 	"github.com/elastic/elastic-agent-libs/logp"
 )
 
+const (
+	ecsEventActionField = "event.action"
+	ecsEventActionValue = "publish-events"
+	ecsEventCountField  = "event.count"
```

Comment:
Where can I find the ECS Event type definition? I've looked at https://www.elastic.co/guide/en/ecs/current/ecs-event.html, but found no mention of `event.count`.

Reply:
Good catch, I think as a practice we don't use ECS fields that do not (yet) exist. A trick we did in the past is to capitalize them to avoid future conflicts, for example `event.Count`.

Reply:
Ack, thanks.

Reply:
Right, `event.count` is not an ECS field.
The reason we used to capitalise non ECS fields is to avoid committing to a schema before it has been officially declared.
In that case, since I didn't find any "counter" field on ECS, I figured I can use `event.count` without capitalizing, because other uses of the same fields (if any) are most likely treat it as a numeric field as well.